CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v0.1.0 (2021-09-22)
+

```

Comment:
@rhdolin It would be great if we can add some nice PR statement here. Do you have some suggestion that would make sense for Bioinformatics world ? 
I had something like below in mind:
"**This our first minor version release with support for blah blah.... For more details follow [Readme](Readme.md). We would love to hear your feedback!
If you find any issue please open an [Issue](link to issue)**".